### ISSUE TITLE
Conflict with AtmosFear

### ISSUE THREAD

**Issue by Pikson** [Decane/SRP#158]
Hello, the new SRP 1.1.4 is causing AtmosFear to corrupt all saves which makes the game stuck on sychronizing, even when starting a new game. Does it mean that I'll have to either play on SRP 1.1.3 or play without AtmosFear?

**Comment by Pikson** [comment]
Yes, I always install SRP first. I have also now updated SRP from 1.1.3 with working atmosfear to 1.1.4 without copying leve_weathers and it is still crashing the game. However after copying the file from 1.1.4, loading is now working properly. So the question is how important is that file for the AtmosFear mod.

**Comment by Decane** [comment]
Beware that any saves created with SRP v1.1.4's level_weathers.script will probably fail to load with AtmosFear's level_weathers.script (but not vice-versa).

**Comment by Pikson** [comment]
> If you've upgraded your working SRP v1.1.3 + AtmosFear installation to SRP v1.1.4 without re-installing AtmosFear afterward, you've probably broken some aspects of AtmosFear's functionality. My advice would be to re-install AtmosFear, letting it overwrite any conflicting files in the SRP, including level_weathers.script.

Yeah, I did it also in this order a few times before. In conclusion, the game doesn't crash only with level_weathers from SRP1.1.4

**Comment by Decane** [comment]
Per [this Reddit thread](https://www.reddit.com/r/stalker/comments/18kiew9/stalker_cszrpatmosfearabsolutenature_crashes_on/), one solution is to use https://pastebin.com/XvWtj3rW as your level_weathers.script. It works with SRP v1.1.4 but I cannot guarantee that it will also work with future versions. By using it, you forfeit the following SRP mitigation:

>Mitigated the issue where the weather would sometimes inexplicably change after loading.
